src/firmware/guest/types/mod.rs

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Currently, the `firmware/guest` module is only for SEV-SNP. This should be changed to allow users to include a legacy SEV attestation report type, correct?

Reply:
Correct, that is what this would be doing. That being said, I wasn't sure where we should put this, as it is a guest-level action which will be consumed in `sevctl`. We could potentially just put it in the `host` module, but it isn't really a host action. I couldn't remember. Are we splitting it by device driver or by use-case?

Reply:
We are re-exporting it into the types mod anyway. This shouldn't be an issue.